docs/0002-nixos-render-docs-Support-gfm-style-admonitions.patch

```diff
@@ -0,0 +1,112 @@
+From 50c0e6b6075b8011e7643bf4fcce710030a72dd8 Mon Sep 17 00:00:00 2001
+From: Quentin Boyer <quentin+dev@familleboyer.net>
+Date: Wed, 11 Sep 2024 19:21:12 +0200
+Subject: [PATCH 2/2] nixos-render-docs: Support gfm-style admonitions
```

Comment:
I know this is unimportant, but:

Since this is now 2/2 can you update my patch too to be part of the patchset?

If you do that, maybe reword one of the commits so either none or both use the `nixos-render-docs: ` prefix?

Reply:
I don't know how to apply your patch in git, I use `patch -p1`, so it would reset all the metadata :/

Reply:
```
git am --directory pkgs/**/nixos-render-docs/src ~/nixvim/main/docs/0001-Output-GFM-admonition.patch
```